.gitattributes

```diff
@@ -29,12 +29,14 @@ zap.pl           crlf=input
 
 # Custom attribute to mark sources as using our C++/C code style.
 [attr]our-c-style  whitespace=tab-in-indent,no-lf-at-eof  hooks.style=KWStyle,clangformat
+[attr]our-py-style whitespace=tab-in-indent,no-lf-at-eof  format.flake8
```

Comment:
Comment above should be changed from 

```
Custom attribute to mark sources as using our C++/C code style.
```

to 

```
Custom attributes to mark sources as using our C++/C/Python/... code styles.
```

Reply:
Also, I suggest:
* `our-py-style` -> `itk-py-style`
* `our-c-style` -> `itk-c-cpp-style`

Reply:
The attribute rename can be done separately; it can't conflict with anything anyways.